_sass/_imla-nama.scss

```diff
@@ -35,6 +35,14 @@ html {
          url("type/FiraSans-Regular.woff") format("woff");
 }
 
+@counter-style urdu-abjad {
+    system: fixed;
```

Comment:
I don't think I have seen looped abjad counting in any urdu book so far? Wish there was a book/document about this. So  I went along with fixed because of https://w3c.github.io/predefined-counter-styles/#arabic-styles

Reply:
Yeah, I'll try finding something about the looped counting. Meanwhile, we can go with `fixed` as it doesn't really matter in our case.